posts/entando-personas-make-modular-applications-more-manageable.md

```diff
@@ -0,0 +1,59 @@
+---
+
+Over the past few years, it has become apparent that businesses need to adapt quickly to changing conditions to stay competitive. Pushing a new feature's time-to-market due to another application part's unpreparedness is unacceptable. Building with a modular architecture in mind can give development teams more independence and potentially improve overall throughput. Unfortunately, modularizing applications can quickly become very complicated and difficult to manage.
+
+## Enter Entando
+Entando is an open-source Application Composition Platform (ACP) that enables companies to build better apps more quickly by facilitating modular application design. Developers and content creators can use the Entando ACP to design, build, and assemble applications from microservices, micro-frontends, and web content. Creating this infrastructure within a Kubernetes environment allows for scalability, availability, and resiliency.
+
+There will always be a diverse set of skills on any given team. Some have UI/UX training. Others see the broader business goals and objectives. It is helpful to have a framework to help everyone get to where they need to be to do the most good. For this, Entando talks about Personas, which provide some guidance and structure to where someone's role may be while working on a composable application.
+
+## What are Personas in Entando?
+Entando defines the Four C's of Application Composition as Create, Curate, Compose, and Consume. Each has a "Persona", or a role that one or more people can fill to focus their strengths towards where they can best contribute. Creators may be more comfortable working in a high-code/traditional-build environment, whereas Composers use a low-code interface to assemble the application. Curators manage and catalog components and identify what may still be needed, and Consumers populate the application with up-to-date and approved web content. 
+
+A single person or team may fit and act as more than one Persona at different times. Still, larger teams may wish to organize people by their technical or business specialties to parallelize the development process better. 
+
+## Creators
+Front-end and back-end developers likely find a Creator role similar to their experience. Creators are responsible for creating and packaging business capabilities, such as a micro-frontend and its associated microservice or a logo widget that can be reused in several of a company's applications, which can then be pushed to a central hub, which may be the official hub hosted by Entando or a internal hub shared throughout a company. A Creator can be a single developer, but there is room for design and business people in this role too! An entire development team consisting of design, development, testing, and business people can keep internally-created components consistent and aligned with business goals. 
```

Comment:
The second sentence is a run-on sentence. I would suggest breaking it into 2 sentences or more.